webapp/src/app/(dashboard)/home/page.tsx

```diff
@@ -55,7 +57,31 @@ export default async function HomePage({
                 <CardHeader>
                     <CardTitle>Get Started</CardTitle>
                     <CardDescription>
-                        Create your first organization to begin
+                        Create your first organization to begin. <br />
+                        You can do that by installing the command line tool and running:
+                        <span style={{
+                            display: "block",
+                            whiteSpace: "pre-wrap",
+                            borderLeft: "2px solid #3498db",
+                            wordWrap: "break-word",
+                            paddingLeft: "1em",
+                            margin: "1em",
+                        }}>
+                        codecarbon login <br />
+                        codecarbon config
+                        codecarbon get-token ...
+                        codecarbon monitor
+                        </span>
+                        You'll then need to get the project id from the config file before generating the token.< br />
```

Comment:
It is not building because of the apostrophe in don't 🤦 . It suggests to use instead `&apos;`
```suggestion
                        You&apos;ll then need to get the project id from the config file before generating the token.< br />
```